subprojects/tooling-api/src/integTest/groovy/org/gradle/integtests/tooling/ToolingApiIntegrationTest.groovy

```diff
@@ -170,7 +172,9 @@ class ToolingApiIntegrationTest extends AbstractIntegrationSpec {
         notThrown(Throwable)
     }
 
+    @IgnoreIf({ isEmbedded() })
```

Comment:
Don't forget to remove the ignore statements after we have a snapshot release with your changes.

Reply:
I got a reminder for that :)